docs/js/remove.md

```diff
@@ -10,31 +10,8 @@ If you really want to get rid of it, you will have to remove its presence from s
 1.  Remove statement `import createSagaMiddleware from 'redux-saga'`.
 2.  Remove statement `const sagaMiddleware = createSagaMiddleware()`.
 3.  Remove `sagaMiddleware` from `middlewares` array.
-4.  Remove statement `store.runSaga = sagaMiddleware.run`
-5.  Remove `store.injectedSagas = {}; // Saga registry`
-
-**app/tests/store.test.js**
-
-1.  Remove describe block and tests for `injectSagas`
-2.  Remove describe block and tests for `runSaga`
-
-**app/utils**
-
-1.  Remove three files: `injectSaga.js`, `sagaInjectors.js`, and `constants.js`.
-
-**app/utils/checkStore.js**
-
-1.  Remove `runSaga: isFunction,`
-2.  Remove `injectedSagas: isObject,`
-
-**app/utils/tests**
-
-1.  Remove two files: `injectSaga.test.js` and `sagaInjectors.test.js`
-
-**app/utils/tests/checkStore.test.js**
-
-1.  Remove `expect(() => checkStore({ ...store, injectedSagas: null })).toThrow();`
-2.  Remove `expect(() => checkStore({ ...store, runSaga: null })).toThrow();`
+4.  Remove statement `const runSaga = sagaMiddleware.run;`
+5.  Remove `runSaga` from `createInjectorsEnhancer` params
```

Comment:
Does this actually work? You can use redux-injectors without redux-saga?

Reply:
kind of. It throws some invariant warnings. I could probably disable them in `redux-injectors`

Reply:
Could it be done by checking which keys are passed to createInjectorsEnhancer? It can be like a configuration object for the injectors and will be especially useful later if/when you add more integrations to them.

Reply:
(No emergency here. We can merge this branch without this being present in injectors.)

Reply:
Possibly, I'd have to look into it more

Reply:
No rush. Just consider it for a future release of injectors.